apps/designer/app/shared/tree-utils/create-instance.ts

```diff
@@ -5,6 +5,10 @@ import {
   components,
 } from "@webstudio-is/react-sdk";
 
+export const createInstanceId = () => {
```

Comment:
Why? to make sure instance id's are created the same way everywhere?

Then we need to update all places that do that as well: there is a couple of them if you search for ObjectId()